package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -344,7 +344,10 @@ cdef class DCDFile:
         cdef int with_unitcell = 1
 
         if isinstance(remarks, six.string_types):
-            remarks = bytearray(remarks, 'ascii')
+            try:
+                remarks = bytearray(remarks, 'ascii')
+            except UnicodeDecodeError:
+                remarks = bytearray(remarks)
```

Comment:
There might be some help in the cython docs.

http://docs.cython.org/en/latest/src/tutorial/strings.html

They have a possible helper function for dealing with string inputs. strings unfortunately are always a nightmare to deal with.

```python
from cpython.version cimport PY_MAJOR_VERSION

cdef unicode _ustring(s):
    if type(s) is unicode:
        # fast path for most common case(s)
        return <unicode>s
    elif PY_MAJOR_VERSION < 3 and isinstance(s, bytes):
        # only accept byte strings in Python 2.x, not in Py3
        return (<bytes>s).decode('ascii')
    elif isinstance(s, unicode):
        # an evil cast to <unicode> might work here in some(!) cases,
        # depending on what the further processing does.  to be safe,
        # we can always create a copy instead
        return unicode(s)
    else:
        raise TypeError(...)
```